mujoco_warp/_src/solver.py

```diff
@@ -893,7 +899,8 @@ def linesearch_parallel_fused(
 
   efc_quad_total_candidate = efc_quad_gauss_in[worldid]
 
-  alpha = float(alphaid) / float(nlsp - 1)
+  alpha = _log_scale(1.0e-6, 1.0, nlsp, alphaid)
```

Comment:
This 1e-6 is a new, rather important constant. Can it be promoted somehow? Given more prominence?

Also I suspect it's too small, but that should really be tested empirically, we should get statistics of the winning alpha, I expect a much larger min value like 1e-3 or 1e-4 makes more sense

Reply:
`min_step` is added to `Option` and a todo is added to determine a default setting for this value.